packages/pds/src/api/app/bsky/actor/getProfile.ts

```diff
@@ -18,38 +18,58 @@ export default function (server: Server) {
       const { ref } = db.db.dynamic
 
       const queryRes = await db.db
-        .selectFrom('user_did')
-        .where(userWhereClause(user))
+        .selectFrom('did_handle')
+        .where(actorWhereClause(actor))
+        .innerJoin(
```

Comment:
inner join because we just want to ignore any repos that don't have a declaration

the thinking here is that a declaration should be required (in our ecosystem) & you're not a "valid `app.bsky` repo" without one

Reply:
Makes sense.  Maybe some day we could offer a method to answer the question "does my repo look alright? if no, what's wrong?"

Reply:
yeah that'd be good fer sure^^